### DISCUSSION TITLE
Script keeps asking me to select media type looping

### DISCUSSION THREAD

**Discussion by wooki0** [DIGITALCRIMINALS/OnlyFans#909] [Q&A]
Not sure if i need to change anything in the config to only select the media for one model at a time and scrape one at a time I feel like it is asking me to select a media type for my entire subscription base am I right to assume this im not going to hit 0 and enter on 1k + subs lol?

**Comment by soulburn74** [comment]
same thing happening here, the current state of things seems to be not quite right, and the 7.6 base install isn't working because of OF's changes to back end doing IP banning, so it seems like we are in a holding pattern of a script that fully works.

**Comment by singularitysky** [comment]
I'm in the same boat.  Finally got it installed and running on OSX, but loops at media type.

7.6 is working for me somewhat but not downloading images, just videos.

**Reply by soulburn74** [reply]
I also revisited 7.6, and depending on how much content a model has, it will sometimes fail with soft IP ban. when they have little enough content, it downloads all content without issue for me. *fingers crossed, and update is released soon that gets past these issues!*
